src/lib.rs

```diff
@@ -14,11 +14,11 @@ The initialisation of a glium display occurs in several steps.
 extern crate glium;
 
 fn main() {
-    // 1. The **winit::EventsLoop** for handling events.
-    let mut events_loop = glium::glutin::event_loop::EventLoop::new();
+    // 1. The **winit::EventLoop** for handling events.
+    let mut events_loop = glium::winit::event_loop::EventLoop::new();
     // 2. Parameters for building the Window.
-    let wb = glium::glutin::window::WindowBuilder::new()
-        .with_inner_size(glium::glutin::dpi::LogicalSize::new(1024.0, 768.0))
```

Comment:
Is this example still accurate?

Reply:
Not really, I've rewritten it using `simple_winit_window`, I hope that's alright, since otherwise it'd get pretty long.

Reply:
Can you add back the function calls here on the `SimpleWindowBuilder`?